src/components/Form.css

```diff
@@ -16,7 +16,8 @@
   padding: 10px;
   padding-bottom: 40px;
   margin-top: 20px;
-  height: 100vh;
+  margin-bottom: 50px;
+  height: 70vh;
```

Comment:
check with marwin about vh, i would use `rem` instead, as VH ignores other rules and applies to the phones sizing and not the other elements or a grid for example.